pallets/identity-management/src/lib.rs

```diff
@@ -264,14 +312,34 @@ pub mod pallet {
 		pub fn identity_activated(
 			origin: OriginFor<T>,
 			account: T::AccountId,
+			id_graph_hash: H256,
 			req_ext_hash: H256,
 		) -> DispatchResultWithPostInfo {
 			let _ = T::TEECallOrigin::ensure_origin(origin)?;
-			Self::deposit_event(Event::IdentityActivated { account, req_ext_hash });
+			IDGraphHash::<T>::insert(account.clone(), id_graph_hash);
+			Self::deposit_event(Event::IdentityActivated { account, id_graph_hash, req_ext_hash });
 			Ok(Pays::No.into())
 		}
 
 		#[pallet::call_index(34)]
+		#[pallet::weight(<T as Config>::WeightInfo::identity_networks_set())]
+		pub fn identity_networks_set(
+			origin: OriginFor<T>,
+			account: T::AccountId,
+			id_graph_hash: H256,
+			req_ext_hash: H256,
+		) -> DispatchResultWithPostInfo {
+			let _ = T::TEECallOrigin::ensure_origin(origin)?;
+			IDGraphHash::<T>::insert(account.clone(), id_graph_hash);
+			Self::deposit_event(Event::IdentityNetworksSet {
+				account,
+				id_graph_hash,
+				req_ext_hash,
+			});
+			Ok(Pays::No.into())
+		}
+
+		#[pallet::call_index(35)]
```

Comment:
Is this harmless update or breaking ?

Reply:
It's a (runtime) breaking change, but it has as much effect on the runtime as the addition of extrinsic parameters. So you'll need to update the parachain runtime anyway